recipes/libgit2/all/conanfile.py

```diff
@@ -103,7 +103,7 @@ def _configure_cmake(self):
         if tools.is_apple_os(self.settings.os):
             cmake.definitions["USE_ICONV"] = self.options.with_iconv
         else:
-            cmake.definitions["USE_ICONV"] = True
+            cmake.definitions["USE_ICONV"] = False
```

Comment:
What I see in the line of code linked by @uilianries  is that for other OSs, this option `USE_ICONV` is not defined, but I agree, setting it to False is equivalent to not defining it